src/components/Tip.tsx

```diff
@@ -22,7 +22,7 @@ export default function Tip({ total, setTip }) {
   const getValidPercentageString = (validTipPercentage: number) => `${validTipPercentage}%`;
 
   return (
-    <>
+    <div>
```

Comment:
in the tests, `wrapper.root.findByType({ component })` is used to help locate the `TextInput` component. If we kept this as a react fragment, when rendering the `root` just points to the first View where all the tip buttons are shown. It does not know that the second View where user can input custom tip exists. 

To resolve this, the react fragment is replaced with a `div` and here's how it looks without the custom tip section and with:
<img width="1023" alt="Screen Shot 2023-07-01 at 9 10 40 PM" src="https://github.com/tally-team/tally-frontend/assets/24977851/4af1a65f-6a51-4d47-be9f-bcb19fe8084c">
<img width="1023" alt="Screen Shot 2023-07-01 at 9 10 46 PM" src="https://github.com/tally-team/tally-frontend/assets/24977851/7d3fa3ff-7c2e-45db-9ad3-44b8155102c7">